robottelo/ui/base.py

```diff
@@ -175,6 +160,21 @@ def delete_entity(self, name, really, name_locator, del_locator,
         strategy, value = del_locator
         self.click((strategy, value % name), wait_for_ajax=False)
         self.handle_alert(really)
+        for _ in range(3):
```

Comment:
Because it is possible that entry will not be deleted in one second (like 'organization' can do). That approach will make our tests more reliable